### ISSUE TITLE
(feat) User comments

### ISSUE THREAD

**Issue by dolanske** [Mavulp/hivefriends#11]
A REST endpoint for storing comments associated to an image key.

A comment could be comprised of:

- User key
- Timestamp
- Text
- Users who liked it

**Comment by dolanske** [comment]
**Idea**

If I send a URL which the back-end can tell is an image / gif. I'd like to receive something like this:

```js
{
   comment: 'You look like a gorilla, just like this one <img src="that_gif_link" />'
}
```

If I send a link which backend deems as a hyperlink, I'd like to receive something like this

```js
{
   comment: 'Woah check me out  <a href="_link_">_link_</a>. My hubby isn't home'
}
```

**Comment by fkaa** [comment]
Added these endpoints:

# `POST /api/images/:image/comments`
### Request
```json
"Hello from Uzbekistan!"
```
### Response
```json
{
    "id": 5,
    "author": "hoblkum",
    "imageKey": "LKaf2-LqMqp",
    "createdAt": 1251951,
    "text": "Hello from Uzbekistan!"
}
```

# `GET /api/images/:image`
### Request
None
### Response
```json
[
    {
        "id": 4,
        "author": "boblam",
        "imageKey": "LKaf2-LqMqp",
        "createdAt": 1251234,
        "text": "Hello everyone!"
    },
    {
        "id": 5,
        "author": "hoblkum",
        "imageKey": "LKaf2-LqMqp",
        "createdAt": 1251951,
        "text": "Hello from Uzbekistan!"
    }
]
```

# `DELETE /api/images/:imagecomments/:comment`
### Request
None
### Response
```json
{
    "id": 5,
    "author": "hoblkum",
    "imageKey": "LKaf2-LqMqp",
    "createdAt": 1251951,
    "text": "Hello from Uzbekistan!"
}
```

**Comment by dolanske** [comment]
Thank you. Will close this when I have it implemented on the front-end too

**Comment by dolanske** [comment]
Using the URL provided in the `DELETE` section gives me 400. I tried doing 

```
https://friends.hivecom.net/api/images/NlFenc9lTViNQB4X1vc2Sw/comments?id=2
```

but that gave me 405. I'd appreciate a bit better description of how to use these functions :/

**Comment by Jokler** [comment]
Fixed by 5fadc26bbd363a2e60fd0fe3edeccd0bec167942.
